RecoParticleFlow/PFClusterProducer/python/particleFlowZeroSuppressionECAL_cff.py

```diff
@@ -12,6 +12,49 @@
 _pfZeroSuppressionThresholds_EEplus_2017 = _pfZeroSuppressionThresholds_EEminus_2017
 
 
+# 
+# The three different set of thresholds will be used to study
+# possible new thresholds of pfrechits and effects on high level objects
+# The values proposed (A, B, C) are driven by expected noise levels
+# A ~ 0.5 sigma noise equivalent thresholds
+# B ~ 1.0 sigma noise equivalent thresholds
+# C ~ 2.0 sigma noise equivalent thresholds
+#
+
+
+# A
+_pfZeroSuppressionThresholds_EB_2018_A = [0.180]*170
+_pfZeroSuppressionThresholds_EEminus_2018_A = [0.22, 0.22, 0.24, 0.26, 0.28, 0.3, 0.32, 0.34, 0.34, 0.36, 0.36, 0.38, 0.38, 0.4, 0.44, 0.46, 0.5, 0.54, 0.58, 0.62, 0.68, 0.72, 0.78, 0.84, 0.9, 1.0, 1.14, 1.36, 1.68, 2.14, 2.8, 3.76, 5.1, 6.94, 9.46, 12.84, 17.3, 23.2, 30.8]
+_pfZeroSuppressionThresholds_EEplus_2018_A = _pfZeroSuppressionThresholds_EEminus_2018_A
+
+_particle_flow_zero_suppression_ECAL_2018_A = cms.PSet(
+    thresholds = cms.vdouble(_pfZeroSuppressionThresholds_EB_2018_A + _pfZeroSuppressionThresholds_EEminus_2018_A + _pfZeroSuppressionThresholds_EEplus_2018_A
+        )
+    )
+
+# B
+_pfZeroSuppressionThresholds_EB_2018_B = [0.140]*170
+_pfZeroSuppressionThresholds_EEminus_2018_B = [0.11, 0.11, 0.12, 0.13, 0.14, 0.15, 0.16, 0.17, 0.17, 0.18, 0.18, 0.19, 0.19, 0.20, 0.22, 0.23, 0.25, 0.27, 0.29, 0.31, 0.34, 0.36, 0.39, 0.42, 0.45, 0.50, 0.57, 0.68, 0.84, 1.07, 1.40, 1.88, 2.55, 3.47, 4.73, 6.42, 8.65, 11.6, 15.4]
```

Comment:
I'm a bit surprised that EE thresholds are lower than EB.
Is it correct?
Do we really need to go to thresholds lower than the "2016 /Run1 style" which has 0.3 GeV in EE?

May I suggest to eventually adjust all thresholds to be *at the minimium* the values of 0.08 in EB and 0.3 in EE.
I don't think we can claim a better performance is still possible by going below the 2016/Run1 state.

Reply:
Indeed, comments have been fixed. Thanks for spotting this.